tests/test_internals/test_movie_interface.py

```diff
@@ -218,3 +218,24 @@ def test_movie_stub(self):
             for no in range(len(metadata["Behavior"]["Movies"])):
                 movie_interface_name = metadata["Behavior"]["Movies"][no]["name"]
                 assert mod[movie_interface_name].data.shape[0] == 10
+
+    def test_movie_timestamps(self):
+        timestamps = list(np.linspace(0.0, 1.16, 30)) + list(np.linspace(50.0, 51.16, 30))
```

Comment:
I love the approach here, in general the tests look great!

One thing, though, that we've been getting more into the habit of doing (since the tests themselves take so long to run) is using only a minimal amount of data in all test cases.

So here, we could simplify the testing timestamps to be simply `timestamps = [1, 2, 4]`

Reply:
Thanks for the feedback! Yeah, I'll change this.